ansible/roles/build-grsec-kernel/defaults/main.yml

```diff
@@ -45,3 +45,9 @@ linux_checksums_url: "{{ linux_base_url }}/sha256sums.asc"
 linux_source_directory: "{{ download_directory }}/linux-{{ linux_kernel_version }}"
 
 gpg_keyserver: hkps.pool.sks-keyservers.net
+
+# Assumes 64-bit (not reading machine architecture dynamically.)
+grsecurity_deb_package: linux-image-{{ linux_kernel_version }}-grsec_{{ linux_kernel_version }}-grsec-10.00.Custom_amd64.deb
+
+grsecurity_use_ccache: true
+ccache_directory: /usr/local/lib/ccache
```

Comment:
Wouldn't somewhere in `/var` be more appropriate? Or somewhere in the home directory of the user that's building the kernels.